prow/plank/controller.go

```diff
@@ -568,3 +571,24 @@ func getPodBuildID(pod *coreapi.Pod) string {
 	logrus.Warningf("BUILD_ID was not found in pod %q: streaming logs from deck will not work", pod.ObjectMeta.Name)
 	return ""
 }
+
+func (c *Controller) patchProwjob(srcPJ prowapi.ProwJob, destPJ prowapi.ProwJob) error {
+	srcPJData, err := json.Marshal(srcPJ)
+	if err != nil {
+		return fmt.Errorf("marshal source prow job: %v", err)
+	}
+
+	destPJData, err := json.Marshal(destPJ)
+	if err != nil {
+		return fmt.Errorf("marshal dest prow job: %v", err)
+	}
+
+	patch, err := jsonpatch.CreateMergePatch(srcPJData, destPJData)
+	if err != nil {
+		return fmt.Errorf("cannot create JSON patch: %v", err)
+	}
+
+	_, err = c.prowJobClient.Patch(srcPJ.Name, ktypes.MergePatchType, patch)
+	c.log.WithFields(pjutil.ProwJobFields(&destPJ)).Debug("Update ProwJob.")
```

Comment:
I tend to prefer present tense for when it will happen (aka if this was on L590) and past tense for when it already happened (current situation)

Reply:
:+1: